functions/go-mail/handler.go

```diff
@@ -0,0 +1,125 @@
+package gomail
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/go-playground/validator/v10"
+	tem "github.com/scaleway/scaleway-sdk-go/api/tem/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+// Body to send to the function (via curl for example or Messaging & Queuing).
+
+//	{
+//		"to": "mail@mail.mail",
+//		"subject": "from console test",
+//		"message": "very very long msg"
+//	}
+
+// region used to call the API.
+const region = scw.RegionFrPar
+
+// Data is a struct to handle the body of the HTTP call. Fields in Data must be completed
+// to send and email.
+type Data struct {
+	Subject string `json:"subject" 	validate:"required"`
+	Message string `json:"message" 	validate:"required"`
+	To      string `json:"to" 		validate:"required,email"`
```

Comment:
```suggestion
	To      string `json:"to" 	validate:"required,email"`
```

Is this code formatted? If not would be good to do a quick gofmt.

Reply:
yes it is 👌 tab vs space here, fixed